src/main/java/frc/robot/Robot.java

```diff
@@ -40,6 +42,21 @@
   Command m_autonomousCommand;
   SendableChooser<Boolean> m_BotChooser = new SendableChooser<>();
 
+  //Inversion Chooser
+  public static SendableChooser<Boolean> m_vliChooser = new SendableChooser<Boolean>();
```

Comment:
The main changes to this class are about adding these 9 boolean dropdowns to the SmartDashboard.

There are 2 dropdowns for each motor on the climber. One inverts the direction of the Talon, the second inverts the phase of the encoder. The idea is to run this on the bot during a brief practice to get the encoders and talons going the correct directions.

The last chooser `m_stateFlag` is just something that you should switch back and forth between true and false every time you change one of the other values. The climber subsystem monitors the `m_stateFlag` variable and updates the settings on the talon every time the `m_stateFlag` chooser changes.